pkg/descheduler/node/node_test.go

```diff
@@ -181,27 +185,53 @@ func TestPodFitsCurrentNode(t *testing.T) {
 					},
 				},
 			},
-			node: &v1.Node{
-				ObjectMeta: metav1.ObjectMeta{
-					Labels: map[string]string{
-						nodeLabelKey: "no",
-					},
-				},
-			},
+			node: test.BuildTestNode("node1", 64000, 128*1000*1000*1000, 200, func(node *v1.Node) {
+				node.ObjectMeta.Labels = map[string]string{
+					nodeLabelKey: "no",
+				}
+
+				node.Status.Allocatable[v1.ResourceEphemeralStorage] = *resource.NewQuantity(1000*1000*1000*1000, resource.DecimalSI)
+			}),
 			success: false,
 		},
 	}
 
+	fakeClient := &fake.Clientset{}
```

Comment:
Does not look to be used anywhere. The `t.Run` redefines the `fakeClient` variable.